recipe/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "isotree" %}
+{% set version = "0.5.24" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d5cb06ac442f3708aa00221f917cb5d18803c9ec256b5224a1cd08935227c9ed
+  patches:
+    - patches/0001-disable-xoshiro.patch
+    - patches/0002-add-unvendored-tsl_robin_map.patch
+
+build:
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+  number: 0
+  missing_dso_whitelist:
+    - '*/libc++.1.dylib'  # [osx]
+    - '*/libomp.dylib'    # [osx]
```

Comment:
```suggestion
```

Reply:
These might be needed, see the last build and its `osx` logs:

- https://cloud.prefect.io/anaconda/flow-run/0a0c8117-8c2f-4ff8-abd0-61ee2196e7f0
- https://cloud.prefect.io/anaconda/flow-run/ccf0acfd-ea74-42f0-8e93-7eacb379ae60

See also for example https://github.com/AnacondaRecipes/scikit-learn-feedstock/blob/a7fca915fa8fadcf9ef9e38f04b87b354f95f317/recipe/meta.yaml#L18-L20

Reply:
Something is terribly wrong if these are missing. When I built the package locally, conda-build was correctly finding these in conda packages. So we should be able to remove them.

Reply:
> Something is terribly wrong if these are missing.

With our build system, or this recipe?

Reply:
Our build system + package ecosystem. Like I said, from what I have seen while building this locally, you should be able to remove these AFAIK.

Reply:
@lorepirri check what these dylibs are and where they should be coming from. Also what problem dso whitelisting is supposed to solve. Give me a shout if you need more info. Google should be good for the first one and the conda-build docs or perseverance skills for the second. That should answer why these shouldn't be needed

Reply:
`missing_dso_whitelist`'s main goal is to warn us that one library (or more) inside the freshly built package links against a library that is not coming from a conda package. When it warns (or errors out in our case), it means that the package will probably be broken if use from another system than the one on which it was built.

Reply:
Or more, `missing_dso_whitelist`'s is to tell conda-build to not complain about ⬆️ in cases where we know what we are doing.

Reply:
https://docs.conda.io/projects/conda-build/en/stable/resources/define-metadata.html#whitelisting-shared-libraries

Reply:
this seems to happen a lot

Reply:
indeed, @beeankha @kenodegard  if there is a ticket for `conda-build` on `missing_dso_whitelist` I will add a link to this PR

Reply:
at first glance I can't see anything. there's another example of this [here](https://github.com/AnacondaRecipes/pytorch-feedstock/blob/ac911a90b8d2d068ca6e8f432b26427808c7189b/recipe/meta.yaml#L69) if you decide to create an issue